snake.js

```diff
@@ -12,6 +12,20 @@ class SnakeGame{
     render() {};
 }
 
+class walls
```

Comment:
please start class number with Uppercase, also name of a class should be singular, eg `Wall`